aquarius/events/events_monitor.py

```diff
@@ -149,11 +150,13 @@ def start_events_monitor(self):
 
     def stop_monitor(self):
         self._monitor_is_on = False
-        if self._updater.is_running():
-            self._updater.stop()
+        if bool(int(os.getenv('PROCESS_POOL_EVENTS', 1)) == 1):
+            if self._updater.is_running():
+                self._updater.stop()
 
     def run_monitor(self):
-        first_update = self._updater.is_first_update_enabled()
+        if bool(int(os.getenv('PROCESS_POOL_EVENTS', 1)) == 1):
```

Comment:
instead of evaluating this multiple times, we can save it in a class attribute then use that in each of the checks